.github/workflows/ghcr-clean.yml

```
@@ -0,0 +1,18 @@
+  delete-untagged-images:
+    name: Delete Untagged Images
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bots-house/ghcr-delete-image-action@v1.1.0
+        with:
+          owner: mendableai
+          name: firecrawl
+          # NOTE: using Personal Access Token
+          token: ${{secrets.GITHUB_TOKEN}}
+          # Keep latest N untagged images
+          untagged-keep-latest: 5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)